src/documents/components/FlatFile/FlatFile.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {Button, Popup} from 'semantic-ui-react';
+import {FlatfileButton} from '@flatfile/react';
+const {parse} = require('json2csv');
+
+/**
+ * FlatFile component for column mapping before document created
+ */
+
+const LICENSE_KEY = 'e118c06a-55ed-47be-90e4-26feb38c3934';
```

Comment:
I think this needs to be read in from the environment and we will soon need a real license key too 😄